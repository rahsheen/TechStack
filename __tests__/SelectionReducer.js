import { test, expect } from 'chai';
import SelectionReducer from '../src/reducers/SelectionReducer'

test('it modifies state appropriately', () => {
    expect(
        SelectionReducer(null,
            {
                type: 'select_library',
                payload: 1000
            }))
        .toMatchSnapshot();
});

