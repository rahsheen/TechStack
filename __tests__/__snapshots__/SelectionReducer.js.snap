// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it modifies state appropriately 1`] = `1000`;
